release_automation.sh

```diff
@@ -52,22 +52,11 @@ fi
 # Check if script is up-to-date
 LOCAL_COMMIT=$(git rev-parse HEAD)
 execute "git" "remote" "update"
-DEVELOP_BRANCH_HEAD=$(git rev-parse 'develop@{upstream}')
-if ! [[ "$LOCAL_COMMIT" = "$DEVELOP_BRANCH_HEAD" ]]; then
-    echo ""
-    echo "You're not running this script from the HEAD commit on the develop branch." 
-    echo "If you are generating a release you should generally use the latest version of the script."
-    read -r -p "Are you sure you want the script to proceed? (y/n) "
-    echo ""
-    if ! [[ $REPLY =~ ^[Yy]$ ]]; then
-        abort "Exiting script..."
-    fi
-fi
```

Comment:
I'm wondering why we would remove this part 🤔. AFAIK this is a safety check in order to assure that we run the script from the latest commit of `develop` branch, right?

Reply:
ah, right, my bad. We should probably rename the default branch on this repo to be `trunk` but we'd still want this check in place.

Reply:
Yep, eventually, the `develop` branch of this repo should also be renamed to `trunk` 😅 .

Reply:
fixed in 32be769a3fcf41ec4ddd4f3bf163ce5b5e59759b